src/.pre-commit-config.yaml

```diff
@@ -11,7 +11,7 @@ repos:
       - id: check-yaml
 
   - repo: https://github.com/psf/black
-    rev: 21.8b0
```

Comment:
Could this be to blame for the tests not passing anymore?

Reply:
Ah! I just notice my comment is outdated and that the tests are passing now. Plus this commit was actually what fixed precommit! 0:)